### PR TITLE
Fix deadlock on Capture finalizing

### PR DESCRIPTION
**Pull request by antonrohr**
http://b/187172673

This is a rather quick fix to not have a deadlock anymore. I filed b/187250643 to clean this up.

### CODE REVIEW

File: src/OrbitGl/App.cpp [hunk 1]
```diff
@@ -1093,6 +1093,8 @@ void OrbitApp::StartCapture() {
       enable_introspection, max_local_marker_depth_per_command_buffer, collect_memory_info,
       memory_sampling_period_ns, std::move(capture_event_processor));
 
+  // TODO (187250643): Refactor this to be more readable and maybe remove parts that are not needed
```
Comment (dpallotti) [inline src/OrbitGl/App.cpp:1096]:
```suggestion
  // TODO (b/187250643): Refactor this to be more readable and maybe remove parts that are not needed
```

**Review by dpallotti — APPROVED** [review]
Thanks!

File: src/OrbitGl/App.cpp (hunk 1 above)
Comment (antonrohr) [reply src/OrbitGl/App.cpp:1096]: done

Comment (dimitry-) [reply src/OrbitGl/App.cpp:1096]: Interestingly enough - if you do http://b/<number> here CLion will show you the status of the bug in the editor.

Comment (dpallotti) [reply src/OrbitGl/App.cpp:1096]: Yep, `b/` is enough to see the status, hence the suggestion.

**Pull request merged**